go.mod

```diff
@@ -5,15 +5,16 @@ go 1.13
 require (
 	github.com/container-storage-interface/spec v1.2.0
 	github.com/docker/go-units v0.4.0
-	github.com/go-logr/logr v0.1.0 // indirect
 	github.com/go-openapi/spec v0.19.4
 	github.com/google/go-cmp v0.4.0 // indirect
 	github.com/google/uuid v1.1.1
+	github.com/jpillora/go-ogle-analytics v0.0.0-20161213085824-14b04e0594ef
```

Comment:
@prateekpandey14 - there as an issue with using this package as is, so in Maya, we had to override with openebs local fork. This mainly happens from clusters that don't have connectivity to external network.

Reply:
Also why not use the github/maya/pkg/usage -- to make sure all the repos are using the same analytics and keywords.

Reply:
Since we are going to deprecate `openebs/maya` repo, i am thinking to move the `usage` pkg to its own repo say `lib-analytics` with keywords types and place holders can be configurable as well, which is kind of blocking us to use it as lib and maintain the copy of similar code in each repo.